### ISSUE TITLE
Integration tests for components using the helper fail

### ISSUE THREAD

**Issue by begedin** [tim-evans/ember-page-title#48]
Since the fix introduced for #44, any integration test for a component which uses the helper internally will fail.

I made an example repository here which demonstrates the failure:
https://github.com/begedin/ember-page-title-bug

The temporary workaround is to do the following in a `beforeEach` for that component's integration test module:

```Javascript
getOwner(this).lookup('router:main').setupRouter();
```

The problem is, the helper does the following at https://github.com/tim-evans/ember-page-title/blob/master/addon/helpers/page-title.js#L34 👍 

```Javascript
let activeTransition = getOwner(this).lookup('router:main').router.activeTransition;
```

However, in integration tests, the router has not been set up, so `getOwner(this).lookup('router:main').router` is `undefined`. A potential solution would be

```Javascript
let router = getOwner(this).lookup('router:main');
let activeTransition = get(router, 'router.activeTransition');
```

**Comment by begedin** [comment]
@tim-evans I see you released 3.1.2 on github but it has not been published on NPM yet. Any chance you could do that?

**Comment by tim-evans** [comment]
Oh yes, sorry!
